### PR TITLE
Adding duplicate_proof capability

### PR DESCRIPTION
**Pull request by pingpingxiu-DOT-ca-gov**
Add staging VDS30SEC's deduplication (with incremental)

### CODE REVIEW

**Comment by pingpingxiu-DOT-ca-gov** [comment]
@ian-r-rose  could you give an approve of this?

File: transform/models/staging/db96/stg_db96__vds30sec.sql [hunk 1]
```diff
@@ -46,3 +52,5 @@ select
     speed_13,
     speed_14
 from {{ source('db96', 'vds30sec') }}
+where true and {{ make_model_incremental('sample_date') }}
```
Comment (ian-r-rose) [inline transform/models/staging/db96/stg_db96__vds30sec.sql:55]: Do we need the `true` condition? I was under the impression that `where {{ make_model_incremental('sample_date') }}` would be enough.

Note that the `make_model_incremental` macro handles the case where there is no other condition already:
https://github.com/cagov/caldata-mdsa-caltrans-pems/blob/78af5cbeffa5158a1302ccd41a7d052faf0fc968/transform/macros/make_model_incremental.sql#L27-L29

**Review by ian-r-rose — COMMENTED** [review]
Looking good to me @pingpingxiu-DOT-ca-gov, just one question

File: transform/models/staging/db96/stg_db96__vds30sec.sql (hunk 1 above)
Comment (pingpingxiu-DOT-ca-gov) [reply transform/models/staging/db96/stg_db96__vds30sec.sql:55]: Thanks @ian-r-rose . indeed the `true` part was not needed. I'm changing it

**Review by kengodleskidot — APPROVED** [review]
No additional comments on my end @pingpingxiu-DOT-ca-gov

**Review by ian-r-rose — APPROVED** [review]
Looks good to me, thanks @pingpingxiu-DOT-ca-gov. I'll merge and do a full refresh of this model today

**Pull request merged**